koku/masu/util/aws/common.py

```diff
@@ -387,46 +387,52 @@ def get_s3_resource():  # pragma: no cover
         aws_secret_access_key=settings.S3_SECRET,
         region_name=settings.S3_REGION,
     )
-    s3_resource = aws_session.resource("s3", endpoint_url=settings.S3_ENDPOINT, config=config)
-    return s3_resource
+    return aws_session.resource("s3", endpoint_url=settings.S3_ENDPOINT, config=config)
 
 
-def copy_data_to_s3_bucket(request_id, path, filename, data, manifest_id=None, context={}):
+def copy_data_to_s3_bucket(request_id, path, filename, data, metadata=None, context=None):
     """
     Copies data to s3 bucket file
     """
-    upload = None
+    if context is None:
```

Comment:
I assume this is because somewhere something is passing `context=None` which sucks. Any chance we could fix that rather than having these if statements for context everywhere? :smile: